snap/gui/google-play-music-desktop-player.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
Can this be moved to `vendor` and somehow indicate that it is related to `snap`.

Something like `vendor/snap/google-play-music-desktop-player.desktop`